LoopUI/Views/LoopCompletionHUDView.swift

```diff
@@ -150,11 +165,18 @@ public final class LoopCompletionHUDView: BaseHUDView {
                     caption?.text = timeString
                 }
 
-                accessibilityLabel = String(format: LocalizedString("Loop ran %@ ago", comment: "Accessbility format label describing the time interval since the last completion date. (1: The localized date components)"), timeString)
-
-                if let fullTimeStr = formatterFull.string(from: ago) {
-                    lastLoopMessage = String(format: LocalizedString("%1$@ last successfully completed a loop %2$@ ago.", comment: "Last loop time completed message (1: app name) (2: last loop time string)"), Bundle.main.bundleDisplayName, fullTimeStr)
+                var timeDateString: String = ""
+                if ago < timeAgoToIncludeDate {
+                    timeDateString = timeFormatter.string(from: date)
+                } else {
+                    timeDateString = timeDateFormatter.string(from: date)
                 }
+
+                let timeString = String(format: LocalizedString("%1$@ ago at \n%2$@", comment: "Format string describing the time interval since and time of the last completion date. (1: The localized date components, (2: the date"), formatter.string(from: ago)!, timeDateString)
```

Comment:
Relative time alone is better for recent Loops, timestamps are more clutter in this case. If you want to put a date string in when last loop is further in the past, that's fine, and at the point we should remove the relative time string.  4 hours is an ok cutoff, imo.

Reply:
See notes below. Alternative code committed.